src/main/java/com/excelsiorjet/api/tasks/JetProject.java

```diff
@@ -893,21 +998,21 @@ private void copyDependency(File from, File to, List<ClasspathEntry> dependencie
      *
      * @return list of dependencies relative to buildDir
      */
-    List<ClasspathEntry> copyDependencies() throws JetTaskFailureException, IOException {
-        File libDir = new File(jetBuildDir, LIB_DIR);
-        Utils.mkdir(libDir);
-        ArrayList<ClasspathEntry> classpathEntries = new ArrayList<>();
+    List<ClasspathEntry> copyClasspathEntries() throws JetTaskFailureException, IOException {
         try {
-            copyDependency(mainJar, new File(jetBuildDir, mainJar.getName()), classpathEntries, false);
-            dependencies.stream()
-                    .filter(a -> a.getFile().isFile())
-                    .forEach(a ->
-                            copyDependency(a.getFile(), new File(libDir, a.getFile().getName()), classpathEntries, a.isLib())
-                    )
-            ;
+            List<ClasspathEntry> classpathEntries = this.classpathEntries.stream()
+                    .filter(a -> a.path.isFile() || a.path.isDirectory())
```

Comment:
It's exists and "is a normal file. A file is normal if it is not a directory and, in addition, satisfies other system-dependent criteria.".
Will change it to isExists